src/webnn_native/ops/Conv2d.cpp

```diff
@@ -52,6 +52,22 @@ namespace webnn_native { namespace op {
         mOptions.dilations = mDilations.data();
         mOptions.dilationsCount = mDilations.size();
 
+        if (options == nullptr || options->outputPadding == nullptr) {
+            mOutputPadding = std::vector<int32_t>(2, 0);
+        } else {
+            mOutputPadding.assign(options->outputPadding,
+                                  options->outputPadding + options->outputPaddingCount);
+        }
+        mOptions.outputPadding = mOutputPadding.data();
+        mOptions.outputPaddingCount = mOutputPadding.size();
+
+        if (options != nullptr && options->outputSizes != nullptr) {
+            mOutputSizes.assign(options->outputSizes,
+                                options->outputSizes + options->outputSizesCount);
+            mOptions.outputSizes = mOutputSizes.data();
+            mOptions.outputSizesCount = mOutputSizes.size();
+        }
+        mOptions.transpose = options == nullptr ? false : options->transpose;
```

Comment:
Should we combine L70-L79 into scope of `if (!options){}`

Reply:
Already optimized. PTAL, thanks!